src/qlik-multi-kpi.less

```diff
@@ -76,38 +76,21 @@
   .value--icon--wrapper{
     display: inline-block;
     position: relative;
-    margin: 0 10px;
-    top: -2px;
-
-    &.mini, &.tiny{
-      top: -4px;
-    }
-    &.small{
-      top: -3px;
-    }
-    &.large{
-      top: 0;
-    }
-    &.huge{
-      top: 3px;
-    }
-  }
-  .value--icon--wrapper.on-value{
-    display: inline-block;
-    position: relative;
-    top: -19px ;
-    &.mini, &.tiny, &.small{
-      top: -21px;
-    }
-
-    &.large{
-      top: -16px;
-    }
-
-    &.huge{
-      top: -15px;
+    margin: 10px;
+    bottom: 3px;
+    &.infographic{
+      margin: 10px 0;
     }
   }
+  // .value--icon--wrapper.on-value{
```

Comment:
Clean up commented code, we can get it from an earlier point in history if we want it back for some reason.